WordPress/gutenberg
# Validation: Present option to convert invalid block to Classic Text

Related: #2132
Related: #974

This pull request seeks to present the user an option to "Convert to Classic Text" for any invalid block. This is an incremental step toward better invalidation resolution, and is not meant to satisfy all requirements of #2132.

![convert](https://user-images.githubusercontent.com/1779930/28931251-74fe6e9c-7843-11e7-9dd7-eb2468e5c5e1.gif)

__Testing instructions:__

1. You'll need an invalid block. Try removing a class name from the markup of the first block in `post-content.js`
2. Navigate to Gutenberg > Demo
3. Note the block modified in step 1 is flagged as invalid and you are presented with a "Convert to Classic Text" button
4. Click "Convert to Classic Text"
5. Note that the block is converted to classic text, and is no longer invalid
